torch/fx/symbolic_trace.py

```diff
@@ -173,6 +173,19 @@ def trace(self, root: Union[torch.nn.Module, Callable]) -> Graph:
         fn, args = self.create_args_for_root(fn, isinstance(root, torch.nn.Module))
 
         orig_call = torch.nn.Module.__call__
+        orig_getattr = torch.nn.Module.__getattr__
+
+        cache = {}
```

Comment:
nit: Probably want to name this something more descriptive like `parameter_proxy_cache`